Content.Client/Corvax/TTS/TTSSystem.cs

```diff
@@ -24,6 +24,16 @@ public sealed class TTSSystem : EntitySystem
     private readonly MemoryContentRoot _contentRoot = new();
     private static readonly ResPath Prefix = ResPath.Root / "TTS";
 
+    /// <summary>
+    /// Reducing the volume of the TTS when whispering. Will be converted to logarithm.
+    /// </summary>
+    public const float WhisperFade = 4f;
+
+    /// <summary>
+    /// The volume at which the TTS sound will not be heard.
+    /// </summary>
+    public const float MinimalVolume = -10f;
```

Comment:
Думаю нет смысла им быть public
```suggestion
    public const float WhisperFade = 4f;

    /// <summary>
    /// The volume at which the TTS sound will not be heard.
    /// </summary>
    public const float MinimalVolume = -10f;
```

Reply:
Оке. Изменил